recipes/proj-data/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.0" %}
+
+package:
+  name: proj-data
+  version: {{ version }}
+
+source:
+  url: http://download.osgeo.org/proj/proj-data-{{ version }}.tar.gz
+  sha256: 2637183bb2fa6cd421085734e5212b0c12550b8d5d3ecaadbfc1044648c7a906
+  folder: data
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - proj >=7.0.0
+
+test:
+  commands:
+    # TODO: we need real tests for each datumgrid.
+    - echo -105 40 | cs2cs +init=epsg:4326 +to +init=epsg:2975
```

Comment:
@snowman2 or @hobu, any input on what would be more meaningful tests?  I didn't see any super obvious examples in the PROJ documentation that make use of these data sets but I also have to admit that I use `proj` through `pyproj` so I don't know base package very well to begin with.

Reply:
If it helps any `cs2cs` is essentially `Transformer.from_crs`. There may be some tests in the pyproj repo and PROJ repo that might be helpful.

Reply:
Thanks, I think I found a decent test in the documentation that I was able to modify for the new `proj-data` file naming and format.